examples/demo.js

```diff
@@ -207,6 +209,39 @@ window.onload = function () {
     });
   }
 
+  //returns the data URL for the last image in the sequence
```

Comment:
Please start with a capital letter, end with a `.` and add space between `//` and the sentence. We are trying to maintain consistency among all comments. Thank you.